### ISSUE TITLE
Subcommand description for `deno test` out of date for 0.33.0 release

### ISSUE THREAD

**Issue by David-Else** [denoland/deno#3999]
```
deno 0.33.0 (Linux, Centos 8.1)
```

In the `--help` docs it says:

```
deno test --help

Searches the specified directories for all files that end in _test.ts or
_test.js and executes them.

  deno test src/
```

but it doesn't work:

```
$ deno test test/
Compile file:///home/david/Downloads/deleteme/.deno.test.ts
error: Uncaught Other: Is a directory (os error 21)
► $deno$/dispatch_json.ts:40:11
    at DenoError ($deno$/errors.ts:20:5)
    at unwrapResponse ($deno$/dispatch_json.ts:40:11)
    at sendAsync ($deno$/dispatch_json.ts:91:10)
$ 
```
I have one file in my `test` directory called `mod_test.ts`

I can run it like this:
```
deno test --allow-all --importmap=./import_map.json test/mod_test.ts
running 1 tests
OK     example test (0.25ms)

test result: OK 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (0.50ms)
```
but total failure trying to run all tests in the directory:
```
$ deno test --allow-all --importmap=./import_map.json test/
Compile file:///home/david/Downloads/deleteme/.deno.test.ts
error: Uncaught Other: Is a directory (os error 21)
► $deno$/dispatch_json.ts:40:11
    at DenoError ($deno$/errors.ts:20:5)
    at unwrapResponse ($deno$/dispatch_json.ts:40:11)
    at sendAsync ($deno$/dispatch_json.ts:91:10)
```

What does this mean when I leave out the permissions and import map info?
```
deno test test/mod_test.ts
relative import path "testing/asserts.ts" not prefixed with / or ./ or ../ Imported from "file:///home/david/Downloads/deleteme/deps.ts"
```

**Comment by bartlomieju** [comment]
Looks like I forgot to update subcommand description 😠 my bad

`deno test` will now run only specified files, so you should rely on your shell glob expansion.

> What does this mean when I leave out the permissions and import map info?
>```
>deno test test/mod_test.ts
>relative import path "testing/asserts.ts" not prefixed with / or ./ or ../ Imported from "file:///home/david/Downloads/deleteme/deps.ts"
>```

That's the error returned when "bare" import specifiers are used (in this case `testing` is the "bare" specifier). 

The error is self explanatory - by default Deno accepts import specifiers starting `/` (absolute), `./`  or `../` (relative). Using "bare" specifiers requires import map.

**Comment by nayeemrmn** [comment]
@David-Else FYI, the old test runner is still available in `std` (and should be until people are satisfied with the new one). That means `deno -A https://deno.land/std@v0.33.0/testing/runner.ts src/` will work like you're used to.

**Comment by David-Else** [comment]
@bartlomieju Cheers! I am very happy with the new `Deno.test()`, and am trying to make a template to share that works like the old `deno test test/` with an import map.

I would like deno to run every test file in the `test` directory. They are titled `xx1_test.ts, xx2_test.ts...`

 I only have one OS here to test on, please can you confirm this is the best way to write it to work on Linux/Mac/Win? I currently have:
```
// import_map.json
{
  "imports": {
    "testing/": "https://deno.land/std@v0.33.0/testing/"
  }
}
```
```
// deps.ts
export { assertEquals } from 'testing/asserts.ts';
```
Now if I use `deno test --allow-all --importmap=./import_map.json test/*` it works and runs all tests. but how can I tell it to only use files that end in `_test`? What is the point of the `_test` if Deno no longer takes it into consideration? Cheers! 

Basically, how do I get the old behavior, that was great, not sure why it changed. I don't mind new syntax, but I am unsure what it should be to work on all OS's:
```
Searches the specified directories for all files that end in _test.ts or
_test.js and executes them.

  deno test src/
```

**Comment by bartlomieju** [comment]
@David-Else this glob should do the trick:
```
deno test --allow-all --importmap=./import_map.json test/**/*_test.{js,ts}
```

**Comment by David-Else** [comment]
@bartlomieju I have this directory structure:
```
├── test
│   ├── mod1_test.ts
│   └── mod_test.ts
```
```
deno test --allow-all --importmap=./import_map.json test/*
```
works, but:
```
deno test --allow-all --importmap=./import_map.json test/**/*_test.{js,ts}

No such file or directory (os error 2)
The terminal process terminated with exit code: 1
```

**Comment by ry** [comment]
I'm trying to fix this in https://github.com/denoland/deno/pull/4002

**Comment by lucacasonato** [comment]
Fixed in #4011
